<testsuite name="specs.cadastro.Cadastro no site de teste" tests="3" errors="0" failures="1" skipped="0" time="14.763116" timestamp="2022-08-30T09:33:34.366042" hostname="CTNDELLGMJCP63"><testcase classname="specs.cadastro.Cadastro no site de teste" name="Realizar cadastro" status="passed" time="4.153614"><system-out>
<![CDATA[
@scenario.begin

  @cadastro
  Cenário: Realizar cadastro
    Dado que acesse o site ... passed in 2.647s
    E que tenha sido gerado um usuario valido ... passed in 0.358s
    E que tenha sido gerado um email valido ... passed in 0.001s
    Dado que acesse o formulario de cadastro ... passed in 0.104s
    Quando preencher as informações necessárias ... passed in 0.145s
    Então o usuario deve ser cadastrado ... passed in 0.899s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase><testcase classname="specs.cadastro.Cadastro no site de teste" name="Realizar cadastro com erro -- @1.1 " status="failed" time="7.453302"><failure type="AssertionError" message="Locator expected to have text 'Nome é obrigatório'&#10;Actual value: None &#10;Call log:&#10;LocatorAssertions.to_have_text with timeout 5000ms&#10;waiting for selector &quot;.alert &gt; span&quot;">
<![CDATA[
Failing step: Então a "Nome é obrigatório" de erro deve ser exibida ... failed in 5.023s
Location: features/specs/cadastro.feature:23
Assertion Failed: Locator expected to have text 'Nome é obrigatório'
Actual value: None 
Call log:
LocatorAssertions.to_have_text with timeout 5000ms
waiting for selector ".alert > span"]]>
</failure><system-out>
<![CDATA[
@scenario.begin

  @cadastro_exception
  Esquema do Cenário: Realizar cadastro com erro -- @1.1 
    Dado que acesse o site ... passed in 1.934s
    E que tenha sido gerado um usuario valido ... passed in 0.178s
    E que tenha sido gerado um email valido ... passed in 0.001s
    Dado que acesse o formulario de cadastro ... passed in 0.176s
    Quando preencher as informações necessárias com "nome nao preenchido" ... passed in 0.141s
    Então a "Nome é obrigatório" de erro deve ser exibida ... failed in 5.023s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase><testcase classname="specs.cadastro.Cadastro no site de teste" name="Realizar cadastro com erro -- @1.2 " status="passed" time="3.156201"><system-out>
<![CDATA[
@scenario.begin

  @cadastro_exception
  Esquema do Cenário: Realizar cadastro com erro -- @1.2 
    Dado que acesse o site ... passed in 1.813s
    E que tenha sido gerado um usuario valido ... passed in 0.159s
    E que tenha sido gerado um email valido ... passed in 0.000s
    Dado que acesse o formulario de cadastro ... passed in 0.153s
    Quando preencher as informações necessárias com "email nao preenchido" ... passed in 0.144s
    Então a "Email é obrigatório" de erro deve ser exibida ... passed in 0.887s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase></testsuite>